### PR TITLE
Fei Protocol Snapshot

### PR DESCRIPTION
**Pull request by Joeysantoro**
#### Hi! What is your PR about?

- [x] Add images for your space
- [x] Add custom domain for your space
- [ ] Add alias for migrate your space
- [x] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: skins/fei.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.my-space {
```
Comment (gokaiorg) [inline skins/fei.scss:1]: The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.fei {
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by sebastiandelgado** [comment]
@gokaiorg Thanks for the suggestion. I just committed it. Just wanted to check that its ok, for the scss file and the css selector to be named `fei` while our directory under `/spaces` is named `/spaces/fei.eth`

**Review by gokaiorg — CHANGES_REQUESTED** [review]
There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?

**Comment by sebastiandelgado** [comment]
@gokaiorg just fixed the merge conflict!

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**